inst/modeldb/PK_2cmt_tvcl_des.R

```diff
@@ -0,0 +1,34 @@
+PK_2cmt_tvcl_des <- function() {
+  description <- "Two compartment PK model with time-dependent clearance using differential equations"
+  ini({
+    lclts <- -1.6 ; label("Time-stationary clearance (CLTS)")
```

Comment:
Please change the name of `lclts` to `lcl` so that it can interoperate with linear models as this appears to be the most similar parameter to the linear clearance in this model.

Reply:
For clarity, please leave the description as "Time-stationary clearance" as that is clearer for what it does in this model.